src/Symfony/Component/Mailer/Bridge/Mailtrap/RemoteEvent/MailtrapPayloadConverter.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Mailer\Bridge\Mailtrap\RemoteEvent;
+
+use Symfony\Component\RemoteEvent\Event\Mailer\MailerDeliveryEvent;
+use Symfony\Component\RemoteEvent\Event\Mailer\MailerEngagementEvent;
+use Symfony\Component\RemoteEvent\Exception\ParseException;
+use Symfony\Component\RemoteEvent\PayloadConverterInterface;
+use Symfony\Component\RemoteEvent\RemoteEvent;
+
+/**
+ * @author Kevin Bond <kevinbond@gmail.com>
+ */
+final class MailtrapPayloadConverter implements PayloadConverterInterface
+{
+    public function convert(array $payload): RemoteEvent
+    {
+        $type = match ($payload['event']) {
+            'delivery' => MailerDeliveryEvent::DELIVERED,
+            'open' => MailerEngagementEvent::OPEN,
+            'click' => MailerEngagementEvent::CLICK,
+            'unsubscribe' => MailerEngagementEvent::UNSUBSCRIBE,
+            'spam' => MailerEngagementEvent::SPAM,
+            'soft bounce', 'bounce' => MailerDeliveryEvent::BOUNCE,
```

Comment:
is it really with a space in the middle? `soft bounce`?

Reply:
Yep

Reply:
thanks, weird 😄